features/capture-stream-canvas.yml

```diff
@@ -0,0 +1,10 @@
+name: captureStream() for <canvas>
+description: The `captureStream()` method for `<canvas>` elements returns a `MediaStream` which includes a `CanvasCaptureMediaStreamTrack` containing a real-time video capture of the contents of the canvas. Firefox doesn't support `CanvasCaptureMediaStreamTrack`, but has `CanvasCaptureMediaStream` which is equivalent.
```

Comment:
Extending my previous suggestion:

```suggestion
description: The `captureStream()` method for `<canvas>` elements returns a `MediaStream` which includes a `CanvasCaptureMediaStreamTrack` representing real-time video of the canvas image. You can use this to record the canvas, or send it elsewhere, such as another canvas or WebRTC connection.
```

One thing I want to note here is that we should endeavor to avoid putting support notes into the description—they should stand the test of time, as browsers resolve their differences. Instead, I think we should deal with this in some sort of notes/partial implementation embellishment. Maybe add a comment like this (wrapped appropriately)?

```
# TODO: Firefox implements `CanvasCaptureMediaStream` equivalent to `CanvasCaptureMediaStreamTrack`. Add a note when https://github.com/web-platform-dx/web-features/issues/915 is resolved. See also: https://bugzil.la/1231131
```

Reply:
sgtm, adding comment